java/arcs/core/host/ArcHostContextParticle.kt

```diff
@@ -119,7 +121,8 @@ class ArcHostContextParticle(
      * is stored in de-normalized format.
      */
     suspend fun readArcHostContext(
-        arcHostContext: arcs.core.host.ArcHostContext
+        arcHostContext: arcs.core.host.ArcHostContext,
+        scheduler: Scheduler
```

Comment:
Isn't the scheduler already available to use via the `EntityHandleManager`? It's not currently present in the `HandleManager` interface def, but maybe it would make sense to expose it... then we could just use that directly here?

Reply:
Added as a fun(). I could have made it a val, but I reasoned we should keep the implementation private (like HandleManager could have a cache or delegate to return them)